src/coreclr/vm/stubmgr.cpp

```diff
@@ -118,6 +119,11 @@ const CHAR * TraceDestination::DbgToString(SString & buffer)
                 pValue = buffer.GetUTF8();
                 break;
 
+            case TRACE_MULTICAST_DELEGATE_HELPER:
+                buffer.Printf("TRACE_MULTICAST_DELEGATE_HELPER(addr=%p)", GetAddress());
```

Comment:
```suggestion
                pValue = "TRACE_MULTICAST_DELEGATE_HELPER";
```
Address isn't used for this case so no need to append (addr=0) to the text.